packages/react-aria-components/src/DropZone.tsx

```diff
@@ -53,12 +54,16 @@ function DropZone(props: DropZoneProps, ref: ForwardedRef<HTMLDivElement>) {
   let {hoverProps, isHovered} = useHover({});
   let {focusProps, isFocused, isFocusVisible} = useFocusRing();
   let stringFormatter = useLocalizedStringFormatter(intlMessages);
+  let isVirtualDrag = isVirtualDragging();
 
   let textId = useSlotId();
+  let dropzoneId = useSlotId();
   let ariaLabel = props['aria-label'] || stringFormatter.format('dropzoneLabel');
-  let messageId = (isDropTarget && props['aria-labelledby']) ? props['aria-labelledby'] : null;
-  let ariaLabelledby = [textId, messageId].filter(Boolean).join(' ');
-  let labelProps = useLabels({'aria-label': ariaLabel, 'aria-labelledby': ariaLabelledby});
+  let messageId = (isVirtualDrag && props['aria-labelledby']) ? props['aria-labelledby'] : null;
```

Comment:
I changed this from `isDropTarget` to `isVirtualDrag` because it wasn't announcing on iOS(). It seemed that because the `aria-labelledby` only pointed to the filled message when `isDropTarget=true` that it was rendering a bit too late for the screenreader to announce. As a result, I changed it to `isVirtualDrag` so that the filled message's `aria-labelledby` is added earlier so the screenreader will announce it. Not sure if this is something we want to include though.

Reply:
Actually, should we even have a conditional on `messageId` here? Don't we want to apply the user provided `aria-labelledby` regardless if it is a virtual drag or not?

Reply:
My thought process was that we only wanted to announce the filled message when it was being displayed which is during a drag. Not sure which is correct though

Reply:
I think having it on there regardless of if its a virtual drag is the way to go. I think if the user is providing it, they would expect it to be applied.

Reply:
Thats a RSP specific implementation detail though right? IMO for RAC Dropzone we should always have the user provided `aria-labelledby` applied since it may be used to label the dropzone in lieu of providing some visible Text as described in our [docs](https://reactspectrum.blob.core.windows.net/reactspectrum/de8064731dc91737cb3fad55de0d4716d21c37d7/docs/react-aria/DropZone.html#anatomy).

Reply:
If we don't want the RSP Dropzone to announce the virtual drop specific message, can we access `let isVirtualDrag = isVirtualDragging();` in the RSP Dropzone? If we can, then you can always provide the user's `aria-labelledby` and only append our own `messageId` for the `isFilled` message to it if `isVirtualDrag` is true